Hazel/src/Platform/OpenGL/OpenGLVertexArray.cpp

```diff
@@ -54,11 +54,16 @@ namespace Hazel {
 		vertexBuffer->Bind();
 
 		uint32_t index = 0;
+		uint32_t count = 0;
```

Comment:
Cherno is quite delicate about using the words size and count.
Maybe something like `indexOffset` or `elementOffset`?

I am not a huge fan of re-calculating this every time we add a vertexBuffer, especialy when add alot of them...
Shouldn't we store this as a member and update the value after adding it to our buffers?
https://github.com/TheCherno/Hazel/blob/f52b813db65cd7275da56003c501914ad1c7515a/Hazel/src/Platform/OpenGL/OpenGLVertexArray.cpp#L70

Reply:
Definitely agreeing with this one. Code review is good as well.